envi/tests/test_arch_arm.py

```diff
@@ -5,58 +5,594 @@
 import envi.registers as e_reg
 import envi.memcanvas as e_memcanvas
 import envi.memcanvas.renderers as e_rend
-import envi.archs.h8 as e_h8
+import envi.archs.arm as arm
 import vivisect
 import platform
-import unittest
+import unittest  
 from envi import IF_RET, IF_NOFALL, IF_BRANCH, IF_CALL, IF_COND
-from envi.archs.h8.regs import *
-from envi.archs.h8.const import *
-from envi.archs.h8.parsers import *
+from envi.archs.arm.regs import *
+from envi.archs.arm.const import *
+from envi.archs.arm.disasm import *
+import binascii   # temporarily included for testing purposes only - allows to print out binary values
 
 
-# OPHEX, VA, repr, flags, emutests
```

Comment:
Note: These were not removed, they were moved... See for reference:

''' 
  This dictionary will contain all commands supported by ARM to test
  Fields will contain following information:
  archBitMask, ophex, va, repr, flags, emutests
'''
I will capitalize them to match

Date: Fri, 3 Jun 2016 05:55:33 -0700
From: notifications@github.com
To: vivisect@noreply.github.com
CC: redheadedrod@hotmail.com; author@noreply.github.com
Subject: Re: [atlas0fd00m/vivisect] Update unit tests for opcodes (#3)

In envi/tests/test_arch_arm.py:

>  
>  
> -# OPHEX, VA, repr, flags, emutests

please put these back in, as they are ARM-specific, and i just added them.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.